Dockerfile.codesniffer

```diff
@@ -0,0 +1,22 @@
+# Stage 1
```

Comment:
Do you think we could reuse the `Dockerfile.ci` container? It does install dev dependencies, so should have the sniffer available.